site/docs/guide/filter-queries.md

```diff
@@ -66,7 +66,7 @@ You can omit some values in the filter queries.
 grammY will then search through different values to match your query.
 
 ```ts
-bot.on(":text"); // all text messages and all text channel posts
+bot.on(":text"); // any text messages and any text post of channels
```

Comment:
```suggestion
bot.on(":text"); // any text messages and any channel posts with text
```
or:
```suggestion
bot.on(":text"); // any text messages and any textual channel posts
```

Reply:
Posts that aren't text posts, can't include text. For example, the photo posts can't have text, but they'll have captions. I think saying text posts is better.

Reply:
What was even wrong with text channel posts? We also have text messages …